### PR TITLE
updates to remove vanilla js

### PR DESCRIPTION
**Pull request by joe-gz**

### CODE REVIEW

**Comment by nolds9** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -20,16 +14,13 @@ Today we will be making a Tic Tac Toe game using all of this.
 1. Construct a `index.html` to be your starting point on this
    project. Add your necessary HTML tags, including `script` and
```
Comment (nolds9) [inline README.md:15]: would recommend changing this to seperate `js` and `css` files. Or is there a specific reason we want them to do it all inline?

Comment (nolds9) [inline README.md:15]: Or at least clarify to have separate files for `html` and `css`

Comment (joe-gz) [inline README.md:15]: @nolds9 there are js and css and css files in the starter code, so they should know to do it separately already?

**Comment by amaseda** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (joe-gz) [inline README.md:15]: changed the wording a bit!

**Pull request merged**